README.md

```diff
@@ -51,12 +52,13 @@
 
 ### Key Features <a name="key-features"></a>
 
-- register and log in, so that the data is private to them.
-- introduce new transactions associated with a category.
-- see the money spent on each category.
+- Authentication - data is private to each user.
+- Makes use of many to many associations - and join tables in the DB
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Live Demo <a name="live-demo"https://rails-budget-app-project.onrender.com</a>
```

Comment:
- [ ] Hi @Ellon-M, great job deploying your app as requested. Unfortunately, I came to realize that there are some differents between the deployed app and the local version. It seems stylings in the deployed app are not being reflected. Let's kindly make sure that our project is properly deployed and that all the pages work as expected. You didn't also use the correct markdown in your readme file and the link appears like this: **Live Demo <a name="live-demo"https://rails-budget-app-project.onrender.com/**. 


| Local version | Deployed version |
| --- | --- |
| ![Screenshot from 2023-04-29 14-59-56](https://user-images.githubusercontent.com/63087491/235306856-af1cb066-3f16-471e-86bf-0ac3253d2f16.png) | ![Screenshot from 2023-04-29 15-00-06](https://user-images.githubusercontent.com/63087491/235307012-dbe3eb3b-f259-4efa-adc4-473ccdfd1f61.png) |

Reply:
Fixed and deployed again!